### PR TITLE
Beta 10

### PR DESCRIPTION
**Pull request by kelseiv**
- update release notes
- beta 10 doc updates

### CODE REVIEW

File: content/v2.0/reference/release-notes/influxdb.md [hunk 1]
```diff
@@ -8,6 +8,23 @@ menu:
 weight: 101
 ---
 
+## v2.0.0-beta.10 [2020-05-07]
+
+### Features
+
+- Add ability to delete a stack and all associated resources.
+- Enforce DNS name compliance on all `pkger` resources' metadata.name field.
```
Comment (sanderson) [inline content/v2.0/reference/release-notes/influxdb.md:16]: Format "metadata.name"  as inline code (`metadata.name`)

Comment (kelseiv) [inline content/v2.0/reference/release-notes/influxdb.md:16]: should fields be plural?

File: content/v2.0/reference/release-notes/influxdb.md [hunk 2]
```diff
@@ -8,6 +8,23 @@ menu:
...
+
+### Bug Fixes
+
+- Ensure `UpdateUser` cleans up the index when updating names.
+- Ensure Checks can be set for zero values.
+
+### UI Improvements
+
+- Create buckets in the Data Explorer and Cell Editor.
+
```
Comment (sanderson) [inline content/v2.0/reference/release-notes/influxdb.md:27]: Add a horizontal rule after each version in the release notes to add visual separation.

```md
- Create buckets in the Data Explorer and Cell Editor.

---

## v2.0.0-beta.9 [2020-04-23]
```

File: content/v2.0/reference/release-notes/influxdb.md (hunk 1 above)
Comment (sanderson) [reply content/v2.0/reference/release-notes/influxdb.md:16]: Yes, probably.

Comment (jsteenb2) [reply content/v2.0/reference/release-notes/influxdb.md:16]: 🤔 the field that has to be compliant is singular (`metadata.name` field), but is enforced for all resources. I'm not sure what the correct grammar is now that I'm on the spot 😬

Comment (kelseiv) [reply content/v2.0/reference/release-notes/influxdb.md]: Thanks, guys - @jsteenb2 would this be accurate?:
Enforce DNS name compliance on the `metadata.name` field in all `pkger` resources.

Comment (jsteenb2) [reply content/v2.0/reference/release-notes/influxdb.md]: smashing, that is spot on 👏

**Pull request merged**